rqt_bag_plugins/src/rqt_bag_plugins/plot_view.py

```diff
@@ -448,9 +443,9 @@ def _add_msg_object(self, parent, path, name, obj, obj_type):
             subobjs = []
 
         plotitem = False
-        if type(obj) in [int, long, float]:
+        if type(obj) in [int, float]:
             plotitem = True
-            if type(obj) == float:
+            if type(obj) is float:
```

Comment:
```suggestion
            if isinstance(obj, float):
```
my suggestion on comparing datatype.

Reply:
Good call, fixed in [424eab6](https://github.com/ros-visualization/rqt_bag/pull/171/commits/424eab6c6a5b7f81abe260f164f64af4e45657ed).